website/docs/d/pi_dhcp.html.markdown

```diff
@@ -34,7 +34,7 @@ In addition to all argument reference list, you can access the following attribu
   Nested scheme for `leases`:
   - `instance_ip` - (String) The IP of the PVM Instance.
   - `instance_mac` - (String) The MAC Address of the PVM Instance.
-- `network` - (String) The ID of the DHCP Server private network (deprecated - replaced by `network_id`).
+- `network` - (Deprecated, String) The ID of the DHCP Server private network.
```

Comment:
For the markdown the original fit much better here than your change since it providing the field it has been replaced by.
```suggestion
- `network` - (Deprecated, String) The ID of the DHCP Server private network ; replaced by `network_id`.
```
If we are not returning it anymore, we might remove altogether as it has been deprecated for a while.

Reply:
Removed deprecated fields.